src/security-rules/security-rules-api-client.ts

```diff
@@ -19,7 +19,25 @@ import { PrefixedFirebaseError } from '../utils/error';
 import { FirebaseSecurityRulesError, SecurityRulesErrorCode } from './security-rules-utils';
 import * as validator from '../utils/validator';
 
-const RULES_API_URL = 'https://firebaserules.googleapis.com/v1';
+const RULES_V1_API = 'https://firebaserules.googleapis.com/v1';
+
+export interface Release {
+  readonly name: string;
+  readonly rulesetName: string;
+  readonly createTime: string;
+  readonly updateTime: string;
+}
+
+export interface RulesetContent {
+  readonly source: {
+    readonly files: Array<{name: string, content: string}>;
```

Comment:
it looks like we have a `RulesFile` type, can we make this a `RulesFile[]`?

Reply:
I'd prefer not to import things in `security-rules.ts` into this file. Right now the import chain flows in the other direction. We can probably address this by moving all the shared types into a new file named `security-rules-api.ts`, but that feels like overkill.